supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -329,3 +329,38 @@ See xref:release-notes[Release notes] for guidelines about deprecation notices.
 image:images/no.png[no] We will fix this issue in the 18.3.4 release next February.
 
 image:images/yes.png[yes] It is anticipated that an upcoming release will include a fix for this issue.
+
+[[ip-addresses-and-mac-addresses]]
+== IP addresses and MAC addresses
+
+There are reserved ranges of IP and MAC addresses for documentation purposes. Use them to avoid the likelihood of conflicts and confusion.
+
+[discrete]
+=== Reserved IP address ranges
```

Comment:
I believe that these items are already covered in the IBM style guide, so we wouldn't want to duplicate that guidance here in the SSG.

However if the MAC address piece isn't covered in the ISG, then I think it could make sense to add it in to the SSG.

Can you remove the section on IP addresses and keep just the MAC address piece. Or maybe we could leave this "Reserved IP address ranges" heading and just say to see the ISG for details about the reserved IP addresses to use in docs. Not sure which would be best - others on the council might have an opinion here.

Reply:
I replaced the part about IP addresses with a reference to the IBM Style Guide.